cdap-distributions/src/etc/cdap/conf.dist/cdap-env.sh

```diff
@@ -25,18 +25,21 @@
 # Adds Hadoop and HBase libs to the classpath on startup.
 # If the "hbase" command is on the PATH, this will be done automatically.
 # Or uncomment the line below to point to the HBase installation directly.
-# HBASE_HOME=
+# export HBASE_HOME=
 
 # Extra CLASSPATH locations to add to CDAP Java processes
-# EXTRA_CLASSPATH=""
+# export EXTRA_CLASSPATH=""
 
 # LOCAL_DIR sets the JVM -Duser.dir property of the CDAP processes, and provides a
 # local working directory for application jars during startup if needed
-LOCAL_DIR="/var/tmp/cdap"
+export LOCAL_DIR="/var/tmp/cdap"
 
 # TEMP_DIR sets the JVM -Djava.io.tmpdir property of the CDAP processes, and provides
 # temporary storage for Apache Twill
-TEMP_DIR="/tmp"
+export TEMP_DIR="/tmp"
+
+# Create heapdump upon OutOfMemoryError (true by default)
```

Comment:
since the location is not configurable (would be nice to add in the future), maybe just a note here that the CDAP log directory will be used.